src/app/index.ts

```diff
@@ -295,7 +295,7 @@ module.exports = class extends yo {
           ? answerForHost.host
           : this.options.host
           ? this.options.host
-          : jsonData.getHostTemplateNames(projType)[0],
+          : jsonData?.getHostTemplateNames(projType)[0],
```

Comment:
It would get host = `null` which would error out later on the code in the `npm run convertToSingleHost` line. 
JSON data is currently never null. If it were to be null, then there is an with our code itself not related to a client's path of choices.

In that case, do you think throwing an error if it is null would help? Ideally, this error never appears, but it might happen if we make mistakes in our code in the future.

Reply:
It's probably OK as is.  You are right in that it's a safety for us rather than customers.  I just wanted to make sure there wouldn't be some other cryptic error or silent failure if null was encountered.